pkg/v1/resource/certconfig/desired.go

```diff
@@ -188,7 +223,7 @@ func newCalicoCertConfig(clusterConfig cluster.Config, cert certs.Cert, projectN
 				AllowBareDomains:    false,
 				ClusterComponent:    certName,
 				ClusterID:           clusterConfig.ClusterID,
-				CommonName:          clusterConfig.Domain.Calico,
+				CommonName:          clusterConfig.Domain.API,
```

Comment:
This diff is so confusing now :)

Reply:
Diff was confusing because it was wrong! I edited the calico cert by mistake :( Labels are correct now.

Reply:
LoL, I thought calico doesn't have legacy labels for some reason :D